src/components/FullscreenGrid/styles.module.scss

```diff
@@ -10,15 +12,45 @@
   grid-template-columns: 2fr 5fr 1fr;
   grid-template-rows: min-content 1fr 0px;
   height: 100vh;
+
+  &.hasACollapsedMainSidePanel {
+    grid-template-columns: $expand-button-width 7fr 1fr;
+  }
 }
 
 .Header {
   grid-area: FGHeader;
 }
 
 .mainSidePanel {
+  display: flex;
+  flex-direction: column;
   grid-area: FGMainSidePanel;
+  height: auto;
+  justify-content: space-between;
   margin-left: $default-padding;
+
+  .mainSidePanelToggleButton {
+    height: auto;
```

Comment:
This fixed things in Chrome but there are still a number of quirks that I don't understand. There is at least one bug in Firefox release (not Nightly) that causes unselected accordion items to expand sometimes.

Here's just one example in Nightly (awkward spacing between accordion buttons):

<img width="389" alt="Screenshot 2019-05-22 09 20 43" src="https://user-images.githubusercontent.com/55398/58183227-933fe700-7c74-11e9-9278-70d9e471b989.png">


I tried switching it to a grid and all issues were solved! I see it working great in Firefox (Release, Nightly) and Chrome. I say we use a grid:

```diff
diff --git a/src/components/FullscreenGrid/styles.module.scss b/src/components/FullscreenGrid/styles.module.scss
index 89dd8b2..6156a12 100644
--- a/src/components/FullscreenGrid/styles.module.scss
+++ b/src/components/FullscreenGrid/styles.module.scss
@@ -23,16 +23,16 @@ $expand-button-width: 40px;
 }
 
 .mainSidePanel {
-  display: flex;
+  display: grid;
   flex-direction: column;
   grid-area: FGMainSidePanel;
+  grid-gap: $default-padding;
+  grid-template-rows: 1fr auto;
   height: auto;
-  justify-content: space-between;
   margin-left: $default-padding;
 
   .mainSidePanelToggleButton {
     height: auto;
-    margin-top: $default-padding;
     padding: $default-padding;
     width: 100%;
 
@@ -43,7 +43,7 @@ $expand-button-width: 40px;
 
   &.mainSidePanelIsCollapsed {
     background-color: $light-gray;
-    margin-top: 0;
+    grid-gap: 0;
     padding: 0;
 
     .mainSidePanelToggleButton {

```

Reply:
oh hum, I tried with a lot of content in the accordion tabs because I had issues with that, but I did not retry with short contents :(

Reply:
Thanks for the css patch1